src/main/java/org/broadinstitute/hellbender/bench/LoadRecalTablea.java

```diff
@@ -0,0 +1,36 @@
+package org.broadinstitute.hellbender.bench;
+
+import com.google.cloud.dataflow.sdk.options.PipelineOptions;
+import com.google.cloud.dataflow.sdk.options.PipelineOptionsFactory;
+import com.google.cloud.genomics.dataflow.utils.GCSOptions;
+import org.broadinstitute.hellbender.tools.recalibration.RecalibrationTables;
+import org.broadinstitute.hellbender.utils.dataflow.BucketUtils;
+
+import java.io.IOException;
+import java.io.ObjectInputStream;
+
+/**
+ *
+ */
+public class LoadRecalTablea {
```

Comment:
Make this into a proper hellbender `CommandLineProgram` with the program group in its `CommandLineProgramProperties` annotation set to a new `BenchmarkingProgramGroup`